molgenis-data-platform/src/main/java/org/molgenis/data/platform/config/PlatformConfig.java

```diff
@@ -16,7 +16,8 @@
 
 @Configuration
 @Import({ DataServiceImpl.class, MetaDataServiceImpl.class, EntityManagerImpl.class, RepositoryDecoratorRegistry.class,
-		EntityFactoryRegistry.class, EntityListenersService.class })
+		EntityFactoryRegistry.class, EntityListenersService.class
```

Comment:
If I reformat PlatformConfig with IntelliJ it does not move the caret to the next line. Revert change?